packages/sirius-web/frontend/sirius-web-papaya/package.json

```diff
@@ -41,9 +41,8 @@
     "@eclipse-sirius/sirius-components-trees": "*",
     "@eclipse-sirius/sirius-components-widget-reference": "*",
     "@eclipse-sirius/sirius-web-application": "*",
-    "@material-ui/core": "4.12.4",
-    "@material-ui/icons": "4.11.3",
-    "@material-ui/lab": "4.0.0-alpha.61",
+    "@mui/material": "5.15.19",
```

Comment:
tss-react missing in peer dependencies and mui and tss-react should also be in dev dependencies

Reply:
Done